client/my-sites/site-settings/traffic/index.js

```diff
@@ -6,12 +6,9 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import config from 'config';
 import controller from './controller';
 import mySitesController from 'my-sites/controller';
 
 export default function() {
-	if ( config.isEnabled( 'manage/seo' ) ) {
```

Comment:
Your assumption is correct here - `manage/seo` is enabled everywhere, so it's no longer necessary here. As also mentioned in https://github.com/Automattic/wp-calypso/pull/11788#discussion_r105893089 I'm planning to clean up those keys in a subsequent PR. Will ping you for a review 😉